src/js/angular/sparql-editor/controllers.js

```diff
@@ -68,14 +75,21 @@ function SparqlEditorCtrl($scope,
         };
     };
 
-    $scope.initViewFromUrlParams = () => {
+    // =========================
+    // Private functions
+    // =========================
+    const initViewFromUrlParams = () => {
+        $scope.updateConfig();
         const queryParams = $location.search();
         if (queryParams.hasOwnProperty(RouteConstants.savedQueryName)) {
             // init new tab from shared saved query link
             initTabFromSavedQuery(queryParams);
         } else if (queryParams.hasOwnProperty(RouteConstants.query)) {
             // init new tab from shared query link
             initTabFromSharedQuery(queryParams);
+        } else if (GuidesService.isActive()) {
```

Comment:
I don't quite understand the idea here, but Its kind of quirky to me

Reply:
User guides have steps that explain SPARQL functionality. When the guide is started and the view is open, we always open a new tab to ensure that we don't alter the user's query, as it could be important for them.

Reply:
OK, but why sparql view should know about the existence of the guides? IMO this should not be here

Reply:
Okay. At this moment, the user guides are broken due to the changes in Yasgui. When I start working on the guides, I will refactor this.

Reply:
Can you create a task for it so that we don't forget to remove it from here.

Reply:
Done [GDB-8625](https://ontotext.atlassian.net/browse/GDB-8625)